### ISSUE TITLE
Does not work with modular export 

### ISSUE THREAD

**Issue by PierreAndreis** [blitz-js/babel-plugin-superjson-next#81]
It seems that this plugin does not pick up `getServerSideProps` that are exported as below.
```js
import {getServerSideProps} from "..";

export { getServerSideProps }
export default Page
```

Note that it works fine if we export as shown below, although less preferred
```js
import {getServerSideProps: getServerSidePropsOriginal} from "..";

export const getServerSideProps = getServerSidePropsOriginal
export default Page
```

**Comment by Skn0tt** [comment]
Hi @PierreAndreis! This is caused by how our Babel replacement works, would you be interested in submitting a PR to add support for this?

**Comment by vincaslt** [comment]
I have the same issue when using getServerSideProps as a function declaration, which I think is common usage.

```
export async function getServerSideProps() {
  ...
}
``` 

I changed it to a const arrow function and it started working.

**Comment by Skn0tt** [comment]
Hi @vincaslt! That's surprising, as we do have a test for exactly this declaration:

https://github.com/blitz-js/babel-plugin-superjson-next/blob/87eb281243dceb8f1d7587e247bbe1172bb8a82a/test/pages/gSSP%20function%20declaration/code.js#L1

Could you open a new issue for this, and give some more context on when it happens? A reproduction case would be creat.

**Comment by Skn0tt** [comment]
I have a feeling this could be fixed by #97 (published in v0.4.1), could you verify that?

**Comment by Skn0tt** [comment]
Closing preemptively, feel free to re-open.
